test/externalTests/useChests.js

```diff
@@ -91,10 +101,10 @@ module.exports = () => async (bot) => {
       assert(blockAssert && block2Assert, new Error('The block instance emitted by chestLidMove is not part of the chest oppened'))
       assert.strictEqual(isOpen, 1, new Error('isOpen should be 1 when opened by one only player'))
 
-      setTimeout(() => {
-        bot.removeListener('chestLidMove', handler)
-        chest.close()
-      }, 500)
+      await bot.test.wait(500)
```

Comment:
Unnecessary on 1.17 and wrong on older versions

Reply:
The idea is to keep the chest open for a litle bit to make sure that no more 'chestLidMove' are emitted until it's closed

Reply:
Ahhh, I forgot that it was a time dependent thing. Thanks for the heads up